packages/server/test/unit/security_spec.js

```diff
@@ -0,0 +1,309 @@
+/* eslint-disable
```

Comment:
remove + fix eslint errors

Reply:
would be cool if eslint was hooked up to do this automatically